parlai/agents/transformer/modules.py

```diff
@@ -591,7 +611,7 @@ def __init__(
 
         self.embeddings = embedding
 
-        if self.variant == 'xlm':
+        if self.variant == ['bert', 'xlm']:
```

Comment:
your tests are failing because of this `==`. you need to replace it with `in`

Reply:
-____- Ha, you can really tell this is a WIP. Fixed - thanks!